docker/domserver/Dockerfile

```diff
@@ -3,6 +3,10 @@ LABEL org.opencontainers.image.authors="DOMjudge team <team@domjudge.org>"
 
 ENV DEBIAN_FRONTEND=noninteractive
 
+# Set up user
+RUN groupadd domjudge -g2004
+RUN useradd -m domjudge -u2004 -g2004
+
```

Comment:
I have a draft in branch queue/check-ownership-across-docker-build-stages (f62e173e934a) to check the UID & GID are the same instead of having to hard-code the values.